packages/website-frontend/src/pages/repositories.page.ts

```diff
@@ -0,0 +1,222 @@
+import { Repository } from '@stryker-mutator/dashboard-contract';
+import { LitElement, html, nothing } from 'lit'
+import { customElement, state } from 'lit/decorators.js'
+import { map } from 'lit/directives/map.js';
+import { when } from 'lit/directives/when.js';
+
+
+import { userService } from '../services/user.service';
+import { repositoriesService } from '../services/repositories.service';
+import { organizationsService } from '../services/organizations.service';
+
+const ModalEvents = {
+  toggleRepository: 'toggleRepository',
+  toggleRepositoryClosed: 'toggleRepositoryClosed'
+}
+
+@customElement('stryker-dashboard-repositories-page')
+export class RepositoriesPage extends LitElement {
+  @state()
+  userRepositoryName: string = '';
+
+  @state()
+  repositories: Repository[] = [];
+
+  @state()
+  organizations: { name: string, value: string }[] = [];
+
+  @state()
+  done = { partOne: false, partTwo: false, repositories: true };
+
+  @state()
+  thereAreDisabledRepositories = false;
```

Comment:
`hasDisabledRepositories`? Maybe this could be a getter, rather than a state property you have to keep updating

Reply:
Good suggestions, i could remove a few lines with this :smile: